app/build.gradle

```diff
@@ -29,9 +29,11 @@ android {
     defaultConfig {
         applicationId "com.pileproject.drive"
         minSdkVersion 15
-        targetSdkVersion 22
+        targetSdkVersion 23
```

Comment:
No strong opinion but some say it should be the latest
http://stackoverflow.com/questions/33947660/what-should-i-set-for-compilesdkversion-minsdkversion-and-targetsdkversion